README.md

```diff
@@ -11,7 +11,8 @@ files that implement command plugins.
 #### Pre-requisites
 
 * Go 1.8 or above
-* Linux (for now)
+* Linux
+* Mac OSX
```

Comment:
Full compatibility with osx?

Reply:
I developed it on an OSX machine, so I expect full compatibility so far.